Tools/Frame_params/TradHeli_Copter36_Setup-MP.param

```diff
@@ -63,8 +63,8 @@ PSC_ANGLE_MAX, 0.0
 PSC_POSXY_P, 1
 PSC_POSZ_P, 1
 PSC_VELXY_D, 0.0
-PSC_VELXY_D_FILT, 5
```

Comment:
This param file (Tools/Frame_params/TradHeli_Copter36_Setup-MP.param) also has 3.6 in the name so I don't think it should be changed either.